src/config/index.ts

```diff
@@ -88,3 +89,7 @@ export const EMAIL_CHANGE_EMAIL_ADDRESS_URL = `${HOME_URL}/${EMAIL_CHANGE_EMAIL_
 export const EMAIL_CHECK_ANSWER_URL = `${HOME_URL}/${EMAIL_CHECK_ANSWER}`;
 export const ACCESSIBILITY_STATEMENT_URL = `${HOME_URL}/${ACCESSIBILITY_STATEMENT_PAGE}`;
 export const EMAIL_UPDATE_SUBMITTED_URL = `${HOME_URL}/${EMAIL_UPDATE_SUBMITTED}`;
+export const CHECK_YOUR_ANSWERS_URL = `${HOME_URL}/${CHECK_YOUR_ANSWERS_PAGE}`;
+export const SUBMITTED_URL = "/submitted"; //todo temp for unit testing
```

Comment:
This can come out now as I've added `EMAIL_UPDATE_SUBMITTED_URL`

Reply:
done